experimental/framework/frs-client/src/FrsSimpleUrlResolver.ts

```diff
@@ -0,0 +1,42 @@
+import { IRequest } from "@fluidframework/core-interfaces";
+import { IUrlResolver, IFluidResolvedUrl, IResolvedUrl } from "@fluidframework/driver-definitions";
+import { IUser, ScopeType } from "@fluidframework/protocol-definitions";
+import { generateToken } from "@fluidframework/server-services-client";
+import { FrsConnectionConfig } from "./interfaces";
+
+export class FrsSimpleUrlResolver implements IUrlResolver {
```

Comment:
Is there going to be a different url resolver for FRS?  If yes, what is the "simple" one's role relative to that one (should have a class comment explaining when it's appropriate to choose this one).  If no, then "simple" should be dropped from the name.

Reply:
The main differentiator I was considering here was that we were generating the JWT token on the user's behalf. However, if we can use the ` TokenProvider` from the config as @znewton commented above, I think we can drop the "Simple" here, but I'm waiting to see how we can automatically fetch the key from the `TokenProvider`. Will add a comment regardless to describe what the class is for

Reply:
Ok I've dropped the "Simple" from the naming now that we are always using the `TokenProvider` as supplied by the client, whether that be a generated `InsecureUrlResolver` or one supplied by the developer in the connection config